hieradata/production.yaml

```diff
@@ -91,6 +91,7 @@ govuk::apps::support_api::aws_s3_bucket_name: 'govuk-production-support-api-csvs
 govuk::apps::travel_advice_publisher::enable_email_alerts: true
 govuk::apps::whitehall::cpu_warning: 400
 govuk::apps::whitehall::cpu_critical: 600
+govuk::apps::whitehall::monitoring_ip: "%{hiera('hosts::production::management::hosts.monitoring-1.ip')}"
```

Comment:
I'm unsure whether `.monitoring-1.ip` will work within the hiera function, but we can try that on integration.

Reply:
Also, do we need this line for staging as well? (perhaps it's easier to just put it in `hieradata/common.yaml`)